FSF.Thullo.Api/Controllers/BoardsController.cs

```diff
@@ -3,55 +3,88 @@
 using FSF.Thullo.Core.Services;
 using Microsoft.AspNetCore.Mvc;
 using Microsoft.AspNetCore.Routing;
+using System.Collections.Generic;
 using System.Linq;
 
 namespace FSF.Thullo.Api.Controllers
 {
+  /// <summary>
+  /// This controller provides an interface for working with boards.
+  /// Boards are the root/parent resource to lists and cards.
+  /// </summary>
   [Route("api/Boards")]
   [ApiController]
   public class BoardsController : ControllerBase
   {
     private readonly ThulloService _thulloService;
 
+    /// <summary>
+    /// Constructor for the boards controller.
+    /// </summary>
+    /// <param name="thulloService">Delegation service for Thullo business logic.</param>
     public BoardsController(ThulloService thulloService)
     {
       _thulloService = thulloService;
     }
 
+    /// <summary>
+    /// Gets all boards.
+    /// </summary>
+    /// <returns>A collection of boards.</returns>
     [HttpGet]
-    public IActionResult Get()
+    public ActionResult<List<BoardDto>> Get()
```

Comment:
I think we ought to make these return types `IEnumerable<>`. Don't need to do it for this PR, but we should catch it at some point.